### PR TITLE
Client NearCache invalidation tests should be less timing sensitive

### LINKED ISSUE #7002: ClientMapNearCacheTest.testNearCacheInvalidationWithLFU
```
java.lang.AssertionError: ownedEntryCount should be smaller than 100, but was 100
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.client.map.impl.nearcache.ClientMapNearCacheTest.assertThatOwnedEntryCountIsSmallerThan(ClientMapNearCacheTest.java:610)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.6/769/com.hazelcast$hazelcast-client-legacy/testReport/com.hazelcast.client.map.impl.nearcache/ClientMapNearCacheTest/testNearCacheInvalidationWithLFU/

### PR DESCRIPTION
**Pull request by cangencer**
Fixes #7002

### CODE REVIEW

**Comment by cangencer** [comment]
run-lab-run

File: hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java [hunk 1]
```diff
@@ -767,4 +717,18 @@ protected NearCacheConfig newNoInvalidationNearCacheConfig() {
     protected ClientConfig newClientConfig() {
         return new ClientConfig();
     }
+
+    protected void assertNearCacheInvalidation_whenMaxSizeExceeded(NearCacheConfig config) {
+        final IMap<Integer, Integer> map = getNearCachedMapFromClient(config);
+        populateNearCache(map, MAX_CACHE_SIZE);
```
Comment (ihsandemir) [inline hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java]: Previously this number was MAX_CACHE_SIZE * 2. If that serves a purpose we may have caused a change in what is being tested. What is the affect of this change?

Comment (cangencer) [inline hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java]: the intention of test is to start invalidating entries after the cache is full, so you only need MAX_SIZE for the  cache to be full. The rewritten version fills the cache and then triggers eviction by inserting one more item.

**Comment by ihsandemir** [comment]
:+1:

**Pull request merged**